recipes/ki18n/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ki18n" %}
+{% set version = "5.64.0" %}
+{% set major_minor = ".".join(version.split(".")[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://download.kde.org/stable/frameworks/{{ major_minor }}/ki18n-{{ version }}.tar.xz
+  sha256: ccd2c2f8b14251701f902c9e7d046da1582e544d31edae743911f3554022d024
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    - {{ pin_subpackage('ki18n', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - extra-cmake-modules
```

Comment:
This should be in `build`. Otherwise looks good to me.

Reply:
Thanks!